src/Forms/EffectKaraoke.cs

```diff
@@ -225,10 +228,11 @@ private void PlayAnimation()
             timer1.Start();
         }
 
-        private static double CalculateStepLength(string text, double duration)
+        public static double CalculateStepLength(string text, double duration)

```

Comment:
Made public so Effect TypeWriter and call it (Patch for typewriter will be pull soon)